tasks/framework-tools/package.json

```diff
@@ -5,39 +5,39 @@
   "private": true,
   "type": "module",
   "devDependencies": {
-    "@babel/cli": "7.14.8",
-    "@babel/core": "7.15.0",
-    "@babel/node": "7.14.9",
+    "@babel/cli": "7.15.4",
+    "@babel/core": "7.15.5",
+    "@babel/node": "7.15.4",
     "@babel/plugin-proposal-class-properties": "7.14.5",
-    "@babel/plugin-proposal-decorators": "7.14.5",
+    "@babel/plugin-proposal-decorators": "7.15.4",
     "@babel/plugin-transform-runtime": "7.15.0",
-    "@babel/preset-env": "7.15.0",
+    "@babel/preset-env": "7.15.6",
     "@babel/preset-react": "7.14.5",
     "@babel/preset-typescript": "7.15.0",
-    "@babel/runtime-corejs3": "7.15.3",
+    "@babel/runtime-corejs3": "7.15.4",
     "@redwoodjs/eslint-config": "0.36.4",
     "@testing-library/jest-dom": "5.14.1",
-    "@testing-library/react": "12.0.0",
+    "@testing-library/react": "12.1.0",
     "@testing-library/user-event": "13.2.1",
-    "@types/jest": "27.0.0",
+    "@types/jest": "27.0.1",
     "ansi-colors": "4.1.1",
-    "babel-jest": "27.0.6",
+    "babel-jest": "27.2.0",
     "babel-plugin-auto-import": "1.1.0",
     "babel-plugin-remove-code": "0.0.6",
-    "core-js": "3.16.1",
+    "core-js": "3.17.3",
     "cp-cli": "2.0.0",
     "cross-env": "7.0.3",
     "fast-glob": "3.2.7",
-    "jest": "27.0.6",
+    "jest": "27.2.0",
     "lerna": "4.0.0",
-    "msw": "0.32.3",
+    "msw": "0.35.0",
     "nodemon": "2.0.12",
-    "npm-packlist": "2.2.2",
+    "npm-packlist": "3.0.0",
```

Comment:
This might mess with rwfw - but I assume you've had a chance to try it?

Reply:
Ah, no I have not other. But I will.

Worst version changelog info ever. But I did look at diff here:
https://github.com/npm/npm-packlist/compare/v2.2.2...v3.0.0

Didn't see anything breaking with API — just package deps.

Reply:
tested locally = ✅